Runtime/Tiles/RuleTile/RuleTile.cs

```diff
@@ -284,7 +284,9 @@ public override bool StartUp(Vector3Int location, ITilemap tilemap, GameObject i
                 Tilemap tmpMap = tilemap.GetComponent<Tilemap>();
                 
                 var iden = Matrix4x4.identity;
-                Quaternion gameObjectQuaternion = new Quaternion();
+                Vector3 gameObjectTranslation = new Vector3();
+                Quaternion gameObjectRotation = new Quaternion();
+                Vector3 gameObjectScale = new Vector3();
```

Comment:
Maybe need to change to ```Vector3 gameObjectScale = Vector3.one;``` to work for default GameObject.

Reply:
Good point.

Reply:
The `orientationMatrix` isn't applied to the default GameObject either.  Let me fix that.